extend-expect.d.ts

```diff
@@ -12,11 +12,12 @@ declare namespace jest {
     toContainHTML(htmlText: string): R
     toHaveAttribute(attr: string, value?: string): R
     toHaveClass(...classNames: string[]): R
+    toHaveFocus(): R
+    toHaveFormValues(object)
```

Comment:
@gnapse I think the reason you were seeing syntax errors is because the parameter needs to be named. As it stands, this is also incorrect syntax.

```ts
toHaveFormValues(values: { [name: string]: any })
```

Reply:
Thanks, I knew I must have been doing something wrong. I guess I need to take a break ;)

Reply:
Also, it should specify a return value of `R`